### ISSUE TITLE
Warp crash when create new tab or new window

### ISSUE THREAD

**Issue by yxunknown** [warpdotdev/Warp#2040]
### Describe the bug

Warp will crash when create new tab or new window.

### To Reproduce

With Warp app open.
1. use shortcut cmd+t create new tab > crash
2. use command palette, input new tab and chose Create New Tab > crash
3. click New Window in right-click menu > crash 

### Expected behaviour

create new tab or new window expectly

### Screenshots

<img width="1361" alt="image" src="https://user-images.githubusercontent.com/22553946/199645159-43a6361d-addc-42dc-98f3-9dc34f2127b1.png">
<img width="585" alt="image" src="https://user-images.githubusercontent.com/22553946/199645197-272ec4fd-dc9f-4ae3-8a28-f689d182d80d.png">

### Operating System

MacOS

### OS Version

macOS Ventura 13.0

### Shell Version

GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22) Copyright (C) 2007 Free Software Foundation, Inc.

### Warp Version

v0.2022.10.25.08.06.stable_03

### Does this block you from using Warp daily?

Yes

**Comment by alokedesai** [comment]
@yxunknown Thanks for reporting. After Warp crashes, are there any error messages in the warp system logs?  To access your logs you can do the following:
* Launch Console, found in Mac’s Applications > Utilities folder
* In the left pane, select `Log Reports`
* In the right pane, find the `warp` log

**Comment by alokedesai** [comment]
Alternately, if there's a corresponding MacOS crash report (located in the `Crash Reports` section of the `Console` app), that would also be very useful

**Comment by yxunknown** [comment]
@alokedesai  Sorry, I can't reproduce this crash today. And follow your tips, I've checked the wapr.log file on my machine:
1. There are lots of [INFO] level logs, no anything about crash info, I think theses logs are unhelpful for you  locating crash;
2. In Crash Reports tab,  no crash reports about Warp;

I updated the macOS ventura in last few days,  it maybe something wrong on the new version system, some other application also crash, like CLion. 

Thanks for you reply and sorry for not giving any helpful info about this problem.
